src/translations/sqlb_nl.ts

```diff
@@ -0,0 +1,7344 @@
+        <source>File %1 already exists. Please choose a different name.</source>
+        <translation>Bestand %1 bestaat al. Kies een andere naam.</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1481"/>
+        <source>Error importing data: %1</source>
+        <translation>Fout bij het importeren van de gegevens: %1</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1483"/>
+        <source>Import completed. Some foreign key constraints are violated. Please fix them before saving.</source>
+        <translation>Importeren voltooid. Sommige vreemde-sleutelbeperkingen werden echter geschonden. Repareer deze voordat je opslaat.</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1485"/>
+        <source>Import completed.</source>
+        <translation>Importeren voltooid.</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1567"/>
+        <source>Delete View</source>
+        <translation>View verwijderen</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1568"/>
+        <source>Modify View</source>
+        <translation>View wijzigen</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1570"/>
+        <source>Delete Trigger</source>
+        <translation>Trigger verwijderen</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1571"/>
+        <source>Modify Trigger</source>
+        <translation>Trigger wijzigen</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1573"/>
+        <source>Delete Index</source>
+        <translation>Index verwijderen</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1574"/>
+        <source>Modify Index</source>
+        <translation>Index wijzigen</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1577"/>
+        <source>Modify Table</source>
+        <translation>Tabel wijzigen</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1624"/>
+        <source>Opened &apos;%1&apos; in read-only mode from recent file list</source>
+        <translation>&apos;%1&apos; geopend vanuit recent-geopende-bestandenlijst in alleen-lezenmodus</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1626"/>
+        <source>Opened &apos;%1&apos; from recent file list</source>
+        <translation>&apos;%1&apos; geopend vanuit recent-geopende-bestandenlijst</translation>
+    </message>
+    <message>
+        <location filename="../MainWindow.cpp" line="1665"/>
+        <source>&amp;%1 %2%3</source>
+        <translation></translation>
```

Comment:
No translation here, but it treated completed. and you can copy and paste the original text for in this case.